ZAPD/GFXDFormatter.cpp

```diff
@@ -0,0 +1,102 @@
+#include "GFXDFormatter.h"
+
+int GFXDFormatter::Write(const char *buf, int count)
```

Comment:
It is OutputFormatter, just renamed. I felt the original name was way too generic, didn't make it clear it was for libgfxd.

Reply:
The intent was to use it on more than just the gfxdis output. Since all it really does is enforce our common formatting patterns like 120 char limit, 4 spaces for tabs etc.
I just never got around to using it on more than display lists

Reply:
Ah, I see. I thought it was something necessary for gfxd to work. My mistake.

Reply:
Yeah its not required at all for libgfxd, glank wrote it in a way that it should be easily used on other parts of the output. Something we can look into soon, cause itll be nice to have the formatting uniform throughout all of our extracted stuff